MockKit/MockPumpManagerState.swift

```diff
@@ -164,6 +164,43 @@ public struct MockPumpManagerState {
         return finalizedDoses + [unfinalizedTempBasal, unfinalizedBolus].compactMap {$0}
     }
 
+    public init(deliverableIncrements: DeliverableIncrements = .medtronicX22,
+                reservoirUnitsRemaining: Double,
```

Comment:
[nit] Consider adding a default here, too. Perhaps getting rid of the constant `pumpReservoirCapacity`, too.